pytket/binders/circuit/main.cpp

```diff
@@ -73,7 +73,8 @@ NB_MODULE(circuit, m) {
       .value("Boolean", EdgeType::Boolean)
       .value("Classical", EdgeType::Classical)
       .value("Quantum", EdgeType::Quantum)
-      .value("WASM", EdgeType::WASM);
+      .value("WASM", EdgeType::WASM)
+      .value("RNG", EdgeType::RNG);
```

Comment:
Is the RNG type only used for the random number function or also for the job ID and job number?

Reply:
Only for the RNG functions. There's no (in-circuit) state associated with the job number functions.